pkg/client/redhatsso/client_test.go

```diff
@@ -800,7 +800,7 @@ func Test_rhSSOClient_UpdateServiceAccount(t *testing.T) {
 				Secret:      serviceAccount.Secret,
 				Name:        &name,
 				Description: &description,
-				OwnerId:     nil,
+				CreatedBy:   nil,
```

Comment:
this change seems unrelated

Reply:
yes - see https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1057